### PR TITLE
ReadTheDocs doc init

### PR DESCRIPTION
**Pull request by robcohen**
*   init the docs for docs.cardano.org

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+
+html_logo = ".sphinx/cardano-logo.png"
+
+html_context = {
+  "display_github": True, # Add 'Edit on Github' link instead of 'View page source'
+  "github_user": "input-output-hk",
+  "github_repo": "cardano-node",
```
Comment (rhyslbw) [inline doc/conf.py:102]:
```suggestion
  "github_repo": "cardano-graphql",
```

File: doc/conf.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    rtd_version = "stable"
+
+# -- Project information -----------------------------------------------------
+
+project = 'cardano-graphql Documentation'
+copyright = '2020, IOHK'
+author = 'IOHK'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0.0'
```
Comment (rhyslbw) [inline doc/conf.py:29]: Is this version meant to be independent of the software version?

File: doc/.sphinx/_templates/layout.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{% extends "!layout.html" %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">YEP THIS IS AN EXAMPLE
```
Comment (rhyslbw) [inline doc/.sphinx/_templates/layout.html:5]: ?

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: doc/.sphinx/_templates/layout.html (hunk 3 above)
Comment (robcohen) [reply doc/.sphinx/_templates/layout.html:5]: This doesn't actually show up, but I can delete it anyway

File: doc/conf.py (hunk 2 above)
Comment (robcohen) [reply doc/conf.py:29]: Yes, the software version is related to specific commits, not something specified in the conf file

**Comment by robcohen** [comment]
@rhyslbw could you re-review?

File: doc/conf.py (hunk 2 above)
Comment (robcohen) [reply doc/conf.py:29]: @rhyslbw I think this is resolved right?

**Comment by rhyslbw** [comment]
@robcohen I'll review this after the immediate priorities, and get it included in the following release

**Pull request closed without merging**